mogi_objects.py

```diff
@@ -386,28 +424,17 @@ async def find_winner(self):
 
             await self.make_teams(winner)
 
-    @discord.ui.button(label="FFA: 0", custom_id="FFA")
-    async def one_button_callback(self, interaction, button):
-        await self.general_vote_callback(interaction, 1, "FFA")
-
-    @discord.ui.button(label="2v2: 0", custom_id="2v2")
-    async def two_button_callback(self, interaction, button):
-        await self.general_vote_callback(interaction, 2, "2v2")
+    def add_button(self, label, callback):
+        button = discord.ui.Button(label=f"{label}: 0", custom_id=label)
+        button.callback = callback
+        self.add_item(button)
```

Comment:
Love it!

Reply:
thanks i wrote something like this for being able to only enable the 6v6 button for rooms above a certain mmr which i did not add into the repo, but it end up coming in handy for this at least